tests/rspec/lib/cluster.rb

```
@@ -192,8 +193,10 @@ def wait_for_bootstrapping
     puts 'HOORAY! The cluster is up'
   rescue
```

This will swallow any error messages that were thrown earlier. Please rescue like: `rescue => e`.

This helped me understand the rescue system of Ruby: https://robots.thoughtbot.com/rescue-standarderror-not-exception

---

done